src/data/panel.ts

```diff
@@ -1,13 +1,22 @@
 import { HomeAssistant, PanelInfo } from "../types";
+import { fireEvent } from "../common/dom/fire_event";
 
 /** Panel to show when no panel is picked. */
-const DEFAULT_PANEL = "lovelace";
+export const DEFAULT_PANEL = "lovelace";
 
-export const getDefaultPanelUrlPath = () =>
-  localStorage.defaultPage || DEFAULT_PANEL;
+export const getStorageDefaultPanelUrlPath = () =>
+  localStorage.defaultPanel || DEFAULT_PANEL;
 
-export const getDefaultPanel = (panels: HomeAssistant["panels"]) =>
-  panels[localStorage.defaultPage] || panels[DEFAULT_PANEL];
+export const getDefaultPanelUrlPath = (hass: HomeAssistant) =>
+  hass.defaultPanel || DEFAULT_PANEL;
```

Comment:
We should be able to drop this and have the setter of `hass.defaultPanel` fill this in.